pkg/kubectl/cmd/create/create_clusterrole.go

```diff
@@ -47,7 +48,10 @@ var (
 		kubectl create clusterrole foo --verb=get,list,watch --resource=pods,pods/status
 
 		# Create a ClusterRole name "foo" with NonResourceURL specified
-		kubectl create clusterrole "foo" --verb=get --non-resource-url=/logs/*`))
+		kubectl create clusterrole "foo" --verb=get --non-resource-url=/logs/*
+
+		# Create a ClusterRole name "aggregation" with AggregationRule specified
+		kubectl create clusterrole aggregation --aggregation-rule="rbac.example.com/aggregate-to-monitoring=true"`))
```

Comment:
Nit: change

> kubectl create clusterrole aggregation

To

> kubectl create clusterrole "monitoring"

Just to make sure that this isn't mistaken as an "aggregation" subcommand